src/enclave/App/App.cpp

```diff
@@ -42,10 +42,13 @@
 #include <time.h> // gettimeofday
 #include <string>
 
-#include <sgx_eid.h>     /* sgx_enclave_id_t */
-#include <sgx_error.h>       /* sgx_status_t */
-#include <sgx_uae_service.h>
-#include <sgx_ukey_exchange.h>
+//#include <sgx_eid.h>     /* sgx_enclave_id_t */
+//#include <sgx_error.h>       /* sgx_status_t */
+//#include <sgx_uae_service.h>
+//#include <sgx_ukey_exchange.h>
```

Comment:
We can delete these rather than commenting them out.

Reply:
Removed.